src/mlpack/methods/reinforcement_learning/q_learning_impl.hpp

```diff
@@ -213,6 +213,12 @@ void QLearning<
   ReplayType
 >::TrainCategoricalAgent()
 {
+  arma::colvec params = {2.2145,2.2145,2.2145,2.2145,2.2145,2.2145,2.2145,2.2145,
+                      -0.274,-0.274, 
+                      2.2145,2.2145,2.2145,2.2145,2.2145,2.2145,2.2145,2.2145,
+                      -0.274,-0.274, -0.274,-0.274};
+  learningNetwork.Parameters() = arma::mat(params);
```

Comment:
Either use a rowvec, or `arma::mat(params.t())`, you can verify that the dimension of both match with:

```c++
std::cout << learningNetwork.Parameters().n_rows << std::endl;
std::cout << learningNetwork.Parameters().n_cols << std::endl;

arma::mat temp = arma::mat(params.t());
std::cout << temp.n_rows << std::endl;
std::cout << temp.n_cols << std::endl;
```

Reply:
Nevermind, this was already correct.